driver/api/odbc.cpp

```diff
@@ -796,45 +795,51 @@ SQLRETURN Fetch(
 
 extern "C" {
 
-SQLRETURN SQL_API EXPORTED_FUNCTION_MAYBE_W(SQLConnect)(HDBC connection_handle,
-    SQLTCHAR * dsn,
-    SQLSMALLINT dsn_size,
-    SQLTCHAR * user,
-    SQLSMALLINT user_size,
-    SQLTCHAR * password,
-    SQLSMALLINT password_size) {
-    // LOG(__FUNCTION__ << " dsn_size=" << dsn_size << " dsn=" << dsn << " user_size=" << user_size << " user=" << user << " password_size=" << password_size << " password=" << password);
-
-    return CALL_WITH_HANDLE(connection_handle, [&](Connection & connection) {
-        const auto dsn_str = toUTF8(dsn, dsn_size);
-        const auto user_str = toUTF8(user, user_size);
-        const auto password_str = toUTF8(password, password_size);
-
-        LOG(__FUNCTION__ << " dsn=" << dsn_str << " user=" << user_str << " pwd=" << password_str);
-
-        connection.init(dsn_str, 0, user_str, password_str, "");
+SQLRETURN SQL_API EXPORTED_FUNCTION_MAYBE_W(SQLConnect)(
+    SQLHDBC        ConnectionHandle,
```

Comment:
Pardon my style-nitpicking comments, but:
* why 'CapitalCamelCase' instead of `lower_underscore_case` for arguments name (like for the rest of the code)?
* And why `NameLength1`, `NameLength2`, `NameLength3` instead of `ServerNameLength`, `UserNameLength`, and `AuthernticationLength` respectively ?

Reply:
That's intentional and conscious copy-pasting of ODBC API function prototypes from Microsoft documentations. Everywhere else, in the internal code, I try to use snake_case.